pronotepy/ent.py

```diff
@@ -4,13 +4,16 @@
 
 import requests
 from bs4 import BeautifulSoup
+from urllib.parse import urlparse, parse_qs
+
+from .exceptions import *
 
 log = logging.getLogger(__name__)
 log.setLevel(logging.DEBUG)
 
 
 @typing.no_type_check
-def educonnect(url: str, cookies, username: str, password: str):
+def educonnect(url: str, session, username: str, password: str):
```

Comment:
Does this not have any consequences in other ENT functions?

Reply:
No there is no consequences. I just checked. And Session object contain cookies so it's better to use session.